fe/fe-common/src/main/java/org/apache/doris/catalog/MysqlColType.java

```diff
@@ -21,53 +21,62 @@
 public enum MysqlColType {
-    MYSQL_TYPE_DECIMAL(0, "DECIMAL"),
-    MYSQL_TYPE_TINY(1, "TINY INT"),
-    MYSQL_TYPE_SHORT(2, "SMALL INT"),
-    MYSQL_TYPE_LONG(3, "INT"),
-    MYSQL_TYPE_FLOAT(4, "FLOAT"),
-    MYSQL_TYPE_DOUBLE(5, "DOUBLE"),
-    MYSQL_TYPE_NULL(6, "NULL"),
-    MYSQL_TYPE_TIMESTAMP(7, "TIMESTAMP"),
-    MYSQL_TYPE_LONGLONG(8, "LONGLONG"),
-    MYSQL_TYPE_INT24(9, "INT24"),
-    MYSQL_TYPE_DATE(10, "DATE"),
-    MYSQL_TYPE_TIME(11, "TIME"),
-    MYSQL_TYPE_DATETIME(12, "DATETIME"),
-    MYSQL_TYPE_YEAR(13, "YEAR"),
-    MYSQL_TYPE_NEWDATE(14, "NEWDATE"),
-    MYSQL_TYPE_VARCHAR(15, "VARCHAR"),
-    MYSQL_TYPE_BIT(16, "BIT"),
-    MYSQL_TYPE_TIMESTAMP2(17, "TIMESTAMP2"),
-    MYSQL_TYPE_DATETIME2(18, "DATETIME2"),
-    MYSQL_TYPE_TIME2(19, "TIME2"),
-    MYSQL_TYPE_JSON(245, "JSON"),
-    MYSQL_TYPE_NEWDECIMAL(246, "NEW DECIMAL"),
-    MYSQL_TYPE_ENUM(247, "ENUM"),
-    MYSQL_TYPE_SET(248, "SET"),
-    MYSQL_TYPE_TINY_BLOB(249, "TINY BLOB"),
-    MYSQL_TYPE_MEDIUM_BLOB(250, "MEDIUM BLOB"),
-    MYSQL_TYPE_LONG_BLOB(251, "LONG BLOB"),
-    MYSQL_TYPE_BLOB(252, "BLOB"),
-    MYSQL_TYPE_VARSTRING(253, "VAR STRING"),
-    MYSQL_TYPE_STRING(254, "STRING"),
-    MYSQL_TYPE_GEOMETRY(255, "GEOMETRY"),
-    MYSQL_TYPE_MAP(400, "MAP");
+    MYSQL_TYPE_DECIMAL(0, "DECIMAL", "DECIMAL"),
+    MYSQL_TYPE_TINY(1, "TINYINT", "TINY INT"),
+    MYSQL_TYPE_SHORT(2, "SMALLINT", "SMALL INT"),
+    MYSQL_TYPE_LONG(3, "INTEGER", "INT"),
+    MYSQL_TYPE_FLOAT(4, "FLOAT", "FLOAT"),
+    MYSQL_TYPE_DOUBLE(5, "DOUBLE", "DOUBLE"),
+    MYSQL_TYPE_NULL(6, "NULL", "NULL"),
+    MYSQL_TYPE_TIMESTAMP(7, "TIMESTAMP", "TIMESTAMP"),
+    MYSQL_TYPE_LONGLONG(8, "BIGINT", "LONGLONG"),
+    MYSQL_TYPE_INT24(9, "INT24", "INT24"),
+    MYSQL_TYPE_DATE(10, "DATE", "DATE"),
+    MYSQL_TYPE_TIME(11, "TIME", "TIME"),
+    MYSQL_TYPE_DATETIME(12, "DATETIME", "DATETIME"),
+    MYSQL_TYPE_YEAR(13, "YEAR", "YEAR"),
+    MYSQL_TYPE_NEWDATE(14, "NEWDATE", "NEWDATE"),
+    MYSQL_TYPE_VARCHAR(15, "VARCHAR", "VARCHAR"),
+    MYSQL_TYPE_BIT(16, "BIT", "BIT"),
+    MYSQL_TYPE_TIMESTAMP2(17, "TIMESTAMP2", "TIMESTAMP2"),
+    MYSQL_TYPE_DATETIME2(18, "DATETIME2", "DATETIME2"),
+    MYSQL_TYPE_TIME2(19, "TIME2", "TIME2"),
+    MYSQL_TYPE_JSON(245, "JSON", "JSON"),
+    MYSQL_TYPE_NEWDECIMAL(246, "NEWDECIMAL", "NEW DECIMAL"),
+    MYSQL_TYPE_ENUM(247, "CHAR", "ENUM"),
+    MYSQL_TYPE_SET(248, "CHAR", "SET"),
+    MYSQL_TYPE_TINY_BLOB(249, "TINYBLOB", "TINY BLOB"),
+    MYSQL_TYPE_MEDIUM_BLOB(250, "MEDIUMBLOB", "MEDIUM BLOB"),
+    MYSQL_TYPE_LONG_BLOB(251, "LONGBLOB", "LONG BLOB"),
+    MYSQL_TYPE_BLOB(252, "BLOB", "BLOB"),
+    MYSQL_TYPE_VARSTRING(253, "VARSTRING", "VAR STRING"),
+    MYSQL_TYPE_STRING(254, "CHAR", "STRING"),
+    MYSQL_TYPE_GEOMETRY(255, "GEOMETRY", "GEOMETRY"),
+    MYSQL_TYPE_MAP(400, "MAP", "MAP");
 
-    private MysqlColType(int code, String desc) {
+    private MysqlColType(int code, String typeName, String desc) {
```

Comment:
Should find a more proper name for type and desc. I could not get the difference.

Reply:
In plsql/Var.defineType(), the Plsql Var type will be found through the Mysql type name string.
TODO, supports the correspondence between Doris type and Plsql Var.